src/pages/ViewScholarship.jsx

```diff
@@ -47,6 +49,30 @@ export default function ViewScholarship({ history, location, match }) {
       <Helmet>
         <title>{scholarship.data.name}</title>
       </Helmet>
+
+      {location?.state?.status && (
+        <Alert
```

Comment:
Following up on https://github.com/beyondhb1079/s4us/pull/790#discussion_r772778128 for consistency maybe we can have a `state.alert` object with all the necessary alert properties instead? Then we could refactor and simply render the alert found in `state.alert` (or list of alerts in `state.alerts` if we want to go that route) from `index.js` beneath the **Header**.